_schema.graphql

```diff
@@ -4175,6 +4175,11 @@ type CollectorProfileType {
   self_reported_purchases: String
 }
 
+# Represents either an action or a potential failure
+union CommerceActionOrFailureUnion =
```

Comment:
Question: Do you happen to know how this prefix `Commerce` routes to `Exchange` mutation?

Reply:
The [rename](https://github.com/artsy/metaphysics/blob/c46da5f9c93f7360f8c3ab1c3a9aa97ec03c0112/src/lib/stitching/exchange/schema.ts#L25-L37) happens here I believe.